pom.xml

```diff
@@ -123,30 +123,30 @@
   </pluginRepositories>
 
   <properties>
-    <amqp-10-starter.version>2.3.3</amqp-10-starter.version>
-    <dekorate.version>1.0.3</dekorate.version>
-    <infinispan-starter.version>2.3.3.Final</infinispan-starter.version>
+    <amqp-10-starter.version>2.3.6</amqp-10-starter.version>
```

Comment:
MMMmmm...not sure, shouldn't it be the `2.9.1` --> https://issues.redhat.com/browse/ENTMQCL-2758?

Reply:
I don't think so because that's the product version and here we're only applying the starter.